charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -74,14 +74,16 @@
     path: /etc/passwd
   name: passwd
 {{- end }}
+{{- if or (and (eq (include "should-enable-system-probe" .) "true") .Values.datadog.systemProbe.enableUSM) (and (eq (include "should-enable-security-agent" .) "true") .Values.datadog.securityAgent.compliance.enabled) }}
```

Comment:
Does the `should-enable-system-probe` helper need to be updated to include `datadog.systemProbe.enableUSM`?

Reply:
Good catch 👍